client/flutter/lib/api/user_preferences.dart

```diff
@@ -50,6 +50,17 @@ class UserPreferences {
     return (await SharedPreferences.getInstance())
         .setBool(UserPreferenceKey.NotificationsEnabled.toString(), value);
   }
+  
+  Future<String> getFCMToken() async {
+    return (await SharedPreferences.getInstance())
+            .getString(UserPreferenceKey.FCMToken.toString()) ??
+        "None";
```

Comment:
what's the reason for returning a default value rather than letting this be `null`?

Reply:
It was probably simpler to do this than to declare a nullable type and add the appropriate checks when using it?

Reply:
If so we can use an empty string

Reply:
Done. Thanks.